test/parameters/granule_name.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from 'chai';
+import { hookTransaction } from '../helpers/db';
+import { hookRedirect } from '../helpers/hooks';
+import { hookRangesetRequest } from '../helpers/ogc-api-coverages';
+import hookServersStartStop from '../helpers/servers';
+
+const regularQuery = {
+  format: 'application/x-zarr',
+  granuleName: '001_08_7f00ff_oceania_east',
+};
+
+const wildCardQuery = {
+  format: 'application/x-zarr',
+  granuleName: '001_*',
+};
+
```

Comment:
Maybe add a multiple granuleName values test for completeness.

Reply:
Done.